### PR TITLE
test: add test for vlan.id - v4

### PR DESCRIPTION
**Pull request by AkakiAlice**
Ticket: #1065

Description:
-  Add Suricata-Verify test for vlan.id keyword

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/1065

Previous PR: https://github.com/OISF/suricata-verify/pull/2134

### CODE REVIEW

**Comment by catenacyber** [comment]
Thanks @AkakiAlice I added the label `requires Suricata pr` ;-)

File: tests/detect-vlan-id/test.rules [hunk 1]
```diff
@@ -0,0 +1,2 @@
+alert ip any any -> any any (msg:"Vlan ID is equal to 200 with specific layer"; vlan.id:200,1; sid:1;)
```
Comment (catenacyber) [inline tests/detect-vlan-id/test.rules:1]: Should it be `vlan.id:200,0;` as we see `vlan[0]: 200`

File: tests/detect-vlan-id/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Test for checking the working of vlan.id keyword by creating rules and matching a crafted packet against them. The packet is an ICMP packet with 3 different VLAN ids [200,300,400].
+
+PCAP created with scapy.
```
Comment (catenacyber) [inline tests/detect-vlan-id/README.md:3]: You can include the scapy script in the PR

Comment (jufajardini) [reply tests/detect-vlan-id/README.md:3]: When you do so, please also include the scapy version used, as things sometimes change from version to version

**Pull request closed without merging**